app/models/magazine.rb

```diff
@@ -0,0 +1,4 @@
+class Magazine < ApplicationRecord
+    has_many :subscriptions
+    has_many :subscribers, through: :subscriptions
```

Comment:
Oh, haha. This is what I was talking about in my comment on the other PR! Instead of `has_and_belongs_to_many`, this is the strategy I'm most familiar with out in the "real world" :)